lib/grape/middleware/error.rb

```diff
@@ -92,19 +95,24 @@ def error_response(error = {})
         message = error[:message] || options[:default_message]
         headers = { Grape::Http::Headers::CONTENT_TYPE => content_type }
         headers.merge!(error[:headers]) if error[:headers].is_a?(Hash)
-        backtrace = error[:backtrace] || []
-        rack_response(format_message(message, backtrace), status, headers)
+        backtrace = error[:backtrace] || error[:original_exception] && error[:original_exception].backtrace || []
```

Comment:
here I could replace this with the following:

```
backtrace = error[:backtrace] || error[:original_exception]&.backtrace || []
```

However, rubocop complains because of Ruby 2.1.*. Does Grape still support that Ruby version?